.github/workflows/ci-frontend.yml

```diff
@@ -28,6 +28,6 @@ jobs:
           pip install pytest
           pip install pytest-random-order
           pip install --no-cache-dir .
-      - name: Test flask app
+      - name: Test flask web app
         run: |
-          pytest --random-order asreview/webapp/tests
+          pytest --random-order --ignore=asreview/webapp/tests/integration_tests asreview/webapp/tests
```

Comment:
What about configuring this in `pytest.ini`? It might be annoying for users to run this by default.

Reply:
It didn't work! But maybe I wasn't sharp.